.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
```

Comment:
I'm not sure but there might be a way to test this on specific version expressions like `python>=3.6`.

Reply:
to make sure our python version will handle f-strings